pinball/ui/views.py

```diff
@@ -359,7 +359,7 @@ def signin(request):
 def auth_return(request):
     oauth2_flow = oauth2.OAuth2Flow()
     domains = oauth2_flow.get_domains()
-    flow = oauth2.OAuth2Flow().get_flow(domains[1])
+    flow = oauth2.OAuth2Flow().get_flow(oauth2_flow.domain)
```

Comment:
I think the intention was to extract the domain from the request but I'm guessing since I didn't write this piece of code. @MaoYe I thought you disabled authentication, no?